atc/resource/run_script.go

```diff
@@ -131,8 +131,11 @@ func (resource *resource) runScript(
 				return err
 			}
 		}
-
-		return json.Unmarshal(stdout.Bytes(), output)
+		err := json.Unmarshal(stdout.Bytes(), output)
+		if err != nil {
+			return fmt.Errorf("%s. \nCommand returned this payload:\n %s", err, stdout.String())
```

Comment:
spacing/capitalization feels a little odd, here's a quick suggestion:

```suggestion
			return fmt.Errorf("%s\n\nwhen parsing resource response:\n\n%s", err, stdout.String())
```

```
cannot unmarshal number into go value of type string

when parsing resource response:

[ {"version": 1234 } ]
```